### PR TITLE
Fix rendering of registration.rst

### PR DESCRIPTION
**Pull request by SmritiSatyanV**
Registration.rst page wasn't rendered properly. And changed the usage of 'The user' in every line.
Signed-off-by: SmritiSatyanV <[smriti@union.ai](mailto:smriti@union.ai)>

### CODE REVIEW

File: rsts/concepts/registration.rst [hunk 1]
```diff
@@ -11,24 +11,19 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
- * The user can later use the FlyteAdmin read API's to get details of the execution,
-   monitor it to completion or retrieve a historical execution
- * OR alternatively the user can use the FlyteConsole to visualize the execution in
-   realtime as it progresses or visualize any historical execution. The console
-   also makes it easy to view debugging information for the execution.
- * The user can set specific rules like *notification* on failure or success or
-   publish all events in an execution to a pub-sub system.
- * The user can also query the datastore to get a summary of all the executions and
-   the compute resources consumed.
+* The user defines tasks using the :py:mod:`Flytekit <flytekit:flytekit>` Task Definition language
```
Comment (AlekhyaSasi) [inline rsts/concepts/registration.rst:14]:
```suggestion
* The user defines tasks using the :py:mod:`Flytekit <flytekit:flytekit>` Task Definition language.
```

File: rsts/concepts/registration.rst [hunk 2]
```diff
@@ -11,24 +11,19 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
+* Next, they define a workflow using the :py:mod:`Flytekit <flytekit:flytekit>` Workflow definition language.
+* The user uses flytekit's register CLI to compile the tasks into their serialized representation as described in :std:ref:`Flyte Specification language <flyteidl:flyteidltoc>`. During this, the task representation is bound to a container that constitutes the code for the task. This associated entity is registered with FlyteAdmin using the registerTask API.
```
Comment (AlekhyaSasi) [inline rsts/concepts/registration.rst:16]:
```suggestion
* The user uses Flytekit's register CLI to compile the tasks into their serialized representation as described in :std:ref:`Flyte Specification language <flyteidl:flyteidltoc>`. During this, the task representation is bound to a container that constitutes the code for the task. This associated entity is registered with FlyteAdmin using the registerTask API.
```

File: rsts/concepts/registration.rst [hunk 3]
```diff
@@ -11,24 +11,19 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
+* Next, they use Flytekit's register CLI to compile the workflow into their serialized representation as described in :std:ref:`Flyte Specification language <flyteidl:flyteidltoc>`. The referenced tasks are replaced by their FlyteAdmin registered Identifier, obtained in the previous step. This associated entity is registered with FlyteAdmin using the registerWorkflow API.
```
Comment (AlekhyaSasi) [inline rsts/concepts/registration.rst:17]:
```suggestion
* Next, they use Flytekit's register CLI to compile the workflow into their serialized representation described in :std:ref:`Flyte Specification language <flyteidl:flyteidltoc>`. The referenced tasks are replaced by their FlyteAdmin registered Identifier, obtained in the previous step. This associated entity is registered with FlyteAdmin using the registerWorkflow API.
```

File: rsts/concepts/registration.rst [hunk 4]
```diff
@@ -11,24 +11,19 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
+* Now, they can launch an execution using the FlyteAdmin launch execution API, which requires the necessary inputs provided. This is automatically done if the user uses the Flytectl to launch the execution.
```
Comment (AlekhyaSasi) [inline rsts/concepts/registration.rst:18]:
```suggestion
* They can now launch an execution using the FlyteAdmin launch execution API, which requires the necessary inputs. This is automatically done if the user uses the Flytectl to launch the execution.
```

File: rsts/concepts/registration.rst [hunk 5]
```diff
@@ -11,24 +11,19 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
+* Later, they can use the FlyteAdmin read API's to get details of the execution, monitor it to completion or retrieve a historical execution
```
Comment (AlekhyaSasi) [inline rsts/concepts/registration.rst:19]:
```suggestion
* Later, they can use the FlyteAdmin read APIs to get details of the execution, monitor it to completion, or retrieve a historical execution.
```

File: rsts/concepts/registration.rst [hunk 6]
```diff
@@ -11,24 +11,19 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
+* **OR** can use the FlyteConsole to visualize the execution in real time as it progresses or visualize any historical execution. The console makes it easy to view debugging information for the execution.
```
Comment (AlekhyaSasi) [inline rsts/concepts/registration.rst:20]:
```suggestion
* **OR** can use the FlyteConsole to visualize the execution in real-time as it progresses or visualizes any historical execution. The console makes it easy to view debugging information for execution.
```

Comment (AlekhyaSasi) [inline rsts/concepts/registration.rst:20]: Is this FlyteConsole or Flyteconsole?

File: rsts/concepts/registration.rst [hunk 7]
```diff
@@ -11,24 +11,19 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
+* They can set specific rules such as *notification* on **failure** or **success** or publish all events in an execution to a pub-sub system.
```
Comment (AlekhyaSasi) [inline rsts/concepts/registration.rst:21]:
```suggestion
* They can set specific rules such as *notification* on **failure** or **success** or publish all events in execution to a pub-sub system.
```

File: rsts/concepts/registration.rst [hunk 8]
```diff
@@ -11,24 +11,19 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
+* Lastly, they can also query the datastore to get a summary of all the executions and the compute resources consumed.
 
 Typical Flow Without Flytekit
 -----------------------------
-It is possible to achieve the exact same workflow as above in case Flytekit is not available. Workflows and tasks are purely specifications and can be provided using any tool like ``YAML``, ``JSON``, ``protobuf binary`` or any other programming language. 
+It is possible to achieve the exact same workflow as mentioned above even if Flytekit is not available. Workflows and tasks are purely specifications and can be provided using tools like ``YAML``, ``JSON``, ``protobuf binary`` or any other programming language.
```
Comment (AlekhyaSasi) [inline rsts/concepts/registration.rst:26]:
```suggestion
It is possible to achieve the same workflow as mentioned above even if Flytekit is unavailable. Workflows and tasks are purely specifications and can be provided using tools like ``YAML``, ``JSON``, ``protobuf binary`` or any other programming language. 
```

File: rsts/concepts/registration.rst [hunk 9]
```diff
@@ -8,29 +8,22 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
-   the compute resources consumed.
-
-Typical Flow Without Flytekit
------------------------------
-It is possible to achieve the exact same workflow as above in case Flytekit is not available. Workflows and tasks are purely specifications and can be provided using any tool like ``YAML``, ``JSON``, ``protobuf binary`` or any other programming language. 
-
-Contributions using other tools are welcome.
+Typical Flow 
+-------------
+As a user, the below steps elaborate the registration process.
```
Comment (samhita-alla) [inline rsts/concepts/registration.rst:13]:
```suggestion
The following steps elaborate the registration process:
```

File: rsts/concepts/registration.rst [hunk 10]
```diff
@@ -8,29 +8,22 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
+
+* Define the tasks using the :py:mod:`Flytekit <flytekit:flytekit>` Task Definition language.
+* Define a workflow using the :py:mod:`Flytekit <flytekit:flytekit>` Workflow definition language.
+* Use flytekit's register CLI to compile the tasks into their serialized representation as described in :std:ref:`Flyte Specification language <flyteidl:flyteidltoc>`. During this, the task representation is bound to a container that constitutes the code for the task. This associated entity is registered with FlyteAdmin using the registerTask API.
+* Use flytekit's register CLI to compile the workflow into their serialized representation as described in :std:ref:`Flyte Specification language <flyteidl:flyteidltoc>`. The referenced tasks are replaced by their FlyteAdmin registered Identifier, obtained in the previous step. This associated entity is registered with FlyteAdmin using the registerWorkflow API.
+* Launch an execution using the FlyteAdmin launch execution API, which requires the necessary inputs provided. This is automatically done if the user uses the flytectl to launch the execution.
```
Comment (samhita-alla) [inline rsts/concepts/registration.rst:19]:
```suggestion
* Launch an execution using the FlyteAdmin launch execution API, which requires the necessary inputs provided. This is automatically done if the user uses flytectl to launch the execution.
```

File: rsts/concepts/registration.rst [hunk 11]
```diff
@@ -8,29 +8,22 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
+* Use the FlyteAdmin read APIs to get details of the execution, monitor it to completion, or retrieve a historical execution.
+* **OR** use the FlyteConsole to visualize the execution in real time as it progresses or visualize any historical execution. The console makes it easy to view debugging information for the execution.
+* Set specific rules such as *notification* on **failure** or **success** or publish all events in the execution to a pub-sub system.
+* Query the datastore to get a summary of all the executions and the compute resources consumed.
+
+.. note::
+    It is possible to achieve the exact same workflow as mentioned above even if flytekit is unavailable. Workflows and tasks are purely specifications and can be provided using tools like ``YAML``, ``JSON``, ``protobuf binary`` or any other programming language. Contributions using other tools are welcome.
```
Comment (samhita-alla) [inline rsts/concepts/registration.rst:26]:
```suggestion
    Workflows and tasks are purely specifications and can be provided using tools like ``YAML``, ``JSON``, ``protobuf binary`` or any other programming language, and hence registration is possible using other tools. Contributions welcome!
```

File: rsts/concepts/registration.rst [hunk 12]
```diff
@@ -8,29 +8,22 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
-It is possible to achieve the exact same workflow as above in case Flytekit is not available. Workflows and tasks are purely specifications and can be provided using any tool like ``YAML``, ``JSON``, ``protobuf binary`` or any other programming language. 
-
-Contributions using other tools are welcome.
+Typical Flow 
+-------------
+The following steps elaborate the registration process from the FlyteAdmin point of view:
+
+* Define the tasks using the :py:mod:`Flytekit <flytekit:flytekit>` Task Definition language.
+* Define a workflow using the :py:mod:`Flytekit <flytekit:flytekit>` Workflow definition language.
+* Use `flytectl <https://docs.flyte.org/projects/flytectl/en/latest/gen/flytectl_register_files.html>`__ to compile the tasks into their serialized representation as described in :std:ref:`Flyte Specification language <flyteidl:flyteidltoc>`. During this, the task representation is bound to a container that constitutes the code for the task. This associated entity is registered with FlyteAdmin using the registerTask API.
```
Comment (samhita-alla) [inline rsts/concepts/registration.rst:17]:
```suggestion
* Use `flytectl register CLI <https://docs.flyte.org/projects/flytectl/en/latest/gen/flytectl_register_files.html>`__ to compile the tasks into their serialized representation as described in :std:ref:`Flyte Specification language <flyteidl:flyteidltoc>`. During this, the task representation is bound to a container that constitutes the code for the task. This associated entity is registered with FlyteAdmin using the registerTask API.
```

File: rsts/concepts/registration.rst [hunk 13]
```diff
@@ -8,29 +8,22 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
+* Use flytectl to compile the workflow into their serialized representation as described in :std:ref:`Flyte Specification language <flyteidl:flyteidltoc>`. The referenced tasks are replaced by their FlyteAdmin registered Identifier, obtained in the previous step. This associated entity is registered with FlyteAdmin using the registerWorkflow API.
```
Comment (samhita-alla) [inline rsts/concepts/registration.rst:18]:
```suggestion
* Use flytectl register CLI to compile the workflow into their serialized representation as described in :std:ref:`Flyte Specification language <flyteidl:flyteidltoc>`. The referenced tasks are replaced by their FlyteAdmin registered Identifiers, obtained in the previous step. This associated entity is registered with FlyteAdmin using the registerWorkflow API.
```

File: rsts/concepts/registration.rst [hunk 14]
```diff
@@ -8,29 +8,22 @@ During registration, Flyte validates the workflow structure and saves the workfl
...
-   the compute resources consumed.
-
-Typical Flow Without Flytekit
------------------------------
-It is possible to achieve the exact same workflow as above in case Flytekit is not available. Workflows and tasks are purely specifications and can be provided using any tool like ``YAML``, ``JSON``, ``protobuf binary`` or any other programming language. 
-
-Contributions using other tools are welcome.
+Typical Flow 
+-------------
+The following steps elaborate the registration process from the FlyteAdmin point of view:
```
Comment (samhita-alla) [inline rsts/concepts/registration.rst:13]:
```suggestion
The following steps elaborate on the specifics of the registration process:
```

**Review by samhita-alla — APPROVED** [review]

**Pull request merged**